src/main/java/com/github/javafaker/Faker.java

```diff
@@ -15,6 +15,7 @@
 public class Faker {
     private final RandomService randomService;
     private final FakeValuesService fakeValuesService;
+    Locale locale;
```

Comment:
Why not at least `private`?

Reply:
Because it uses in Address class.

Reply:
Oh sorry, I meant `final`.

Reply:
I don't know how do it without adding new parameter to constructor

```java
public Faker(FakeValuesService fakeValuesService, RandomService random).
```

If you ok I will rewrite it to

```java
public Faker(FakeValuesService fakeValuesService, RandomService random, Locale locale).
```
or add new one?

Reply:
@schallerala which your variant?

Reply:
Sorry for the way too long time for me to answer... What I would do is, change the `FakeValueService` to have a reference of its locales chain and a getter. Then, in the `Faker` class, remove this variable, and replace it with a package private getter, that return the first locale of the locales chain of the value service. 
IMO it would be cleaner than just adding a locale in this class when we handle them already in a _delegate class_.

Reply:
Ok. Looks more pretty. :) Changed.